app/api/teacher.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from typing import Optional
+from typing import List
+from db.db import Database
+from models.models import Queue, Comparator, BriefGroup, Lab
+
+
+class Teacher:
+    def __init__(self, teacher_tg_id: int, db: Database):
+        self.teacher_tg_id = teacher_tg_id
+        self.db = db
+
+    """
+    Creates new empty group.
+    """
+    def create_group(self, group_name: str) -> BriefGroup:
+        pass
+
+    """
+    :return: invitation code of the group.
+    """
+    def get_group_invitation_code(self, group: BriefGroup) -> str:
+        return self.get_group_invite_code(group.id)
+    
+    """
+    :return: invitation code of the group.
+    """
+    def get_group_invitation_code(self, group_id: str) -> str:
+        return group_id
+    
+    """
+    :return: list of all groups.
+    """
+    def get_all_groups(self) -> List[BriefGroup]:
+        pass
+
+    """
+    Adds a new `lab` to the `group.
+    """
+    def add_lab(self, group: BriefGroup, lab: Lab):
+        pass
+
+    """
+    Creates and adds the class with queue for labs pass.
+    """
+    def add_class(self, cls: Queue):
```

Comment:
Aren't class and group the same thing?

Reply:
Yeah, it's the same. I thought it is good idea to call it "class", but it's not.